### DISCUSSION TITLE
Resuming From Lesson 7 Foundry course_23

### DISCUSSION THREAD

**Discussion by AbigirlMuchineripi** [Cyfrin/foundry-full-course-cu#2365] [General]
![image](https://github.com/user-attachments/assets/1b14e74b-1110-42b1-ac2c-493309e92a7a)
Hello everyone , kindly help. It seems as if this course has been updated. I am trying to resume from lesson 7. I am having a challenge on navigating. thank you.

**Comment by EngrPips** [comment]
Hello @AbigirlMuchineripi, why not use https://updraft.cyfrin.io/

**Reply by AbigirlMuchineripi** [reply]
Thanks man.

**Reply by EngrPips** [reply]
You are welcome, Friend.
